spec/requests/admin/podcasts_spec.rb

```diff
@@ -62,15 +62,56 @@
   describe "Updating" do
```

Comment:
A reusable variable for the params to pass in to each test case

```suggestion
  describe "Updating" do
    let(:update_params) do
      {
        title: "hello",
        feed_url: "https://pod.example.com/rss.rss",
        description: "Description",
        itunes_url: "https://itunes.example.com",
        overcast_url: "https://overcast.example.com",
        android_url: "https://android.example.com",
        soundcloud_url: "https://soundcloud.example.com",
        website_url: "https://example.com",
        twitter_username: "@ThePracticalDev",
        pattern_image: fixture_file_upload("files/800x600.png", "image/png"),
        main_color_hex: "ffffff",
        image: fixture_file_upload("files/podcast.png", "image/png")
        slug: "postcast-test-url",
        reachable: true,
        published: true,
      }
    end

```